nWSR: 1000
joint_names:
  - torso_lift_joint
  - l_shoulder_pan_joint
  - l_shoulder_lift_joint
  - l_upper_arm_roll_joint
  - l_elbow_flex_joint
  - l_forearm_roll_joint
  - l_wrist_flex_joint
  - l_wrist_roll_joint
  - r_shoulder_pan_joint
  - r_shoulder_lift_joint
  - r_upper_arm_roll_joint
  - r_elbow_flex_joint
  - r_forearm_roll_joint
  - r_wrist_flex_joint
  - r_wrist_roll_joint
l_arm_names:
  - l_shoulder_pan_joint
  - l_shoulder_lift_joint
  - l_upper_arm_roll_joint
  - l_elbow_flex_joint
  - l_forearm_roll_joint
  - l_wrist_flex_joint
  - l_wrist_roll_joint
r_arm_names:
  - r_shoulder_pan_joint
  - r_shoulder_lift_joint
  - r_upper_arm_roll_joint
  - r_elbow_flex_joint
  - r_forearm_roll_joint
  - r_wrist_flex_joint
  - r_wrist_roll_joint
frame_id: base_link
l_fk_name: left_ee
r_fk_name: right_ee
watchdog_period: 0.1
internals:
  cart_cart:
    doubles: [r_rot_error, l_rot_error, l_trans_error, r_trans_error]
    vectors: []
  cart_joint:
    doubles: [l_rot_error, l_trans_error]
    vectors: []
  joint_cart:
    doubles: [r_rot_error, r_trans_error]
    vectors: []
  joint_joint:
    doubles: []
    vectors: []
